examples/metrics/prometheus-install/prometheus.yaml

```diff
@@ -58,6 +58,9 @@ spec:
   podMonitorSelector:
     matchLabels:
       app: strimzi
+  serviceMonitorSelector:
```

Comment:
the related issue you linked mentions that a "dummy" entry is enough like `serviceMonitorSelector: {}`.
Have you verified it instead of having this match labels entry?

Reply:
No, I can try.

Reply:
Yes please, it would be much better!!

Reply:
Tested and updated.